test

```diff
@@ -50,3 +51,12 @@ if [ -n "${vetRes}" ]; then
 fi
 
 echo "Success"
+
+echo "Checking golint..."
+lintRes=$(golint $LINT)
+if [ -n "${lintRes}" ]; then
+        echo -e "golint checking failed:\n${lintRes}"
+        exit 255
```

Comment:
How about removing the `exit` and just check the logs manually? It won't hurt us as much in case of false positives.